docs/content/development/extensions-core/datasketches-hll.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: doc_page
+---
+
+## DataSketches HLL Sketch module
+
+This module provides Druid aggregators for distinct counting based on HLL sketch from [datasketches](http://datasketches.github.io/) library. At ingestion time, this aggregator creates the HLL sketch objects to be stored in Druid segments. At query time, sketches are read and merged together. In the end, by default, you receive the estimate of the number of distinct values presented to the sketch. Also, you can use post aggregator to produce a union of sketch columns in the same row. 
+You can use the HLL sketch aggregator on columns of any identifiers. It will return estimated cardinality of the column.
+
+To use this aggregator, make sure you [include](../../operations/including-extensions.html) the extension in your config file:
+
+```
+druid.extensions.loadList=["druid-datasketches"]
+```
+
+### Aggregators
+
+```json
```

Comment:
I don't think these should be marked as `json` since it isn't technically valid json with the `<` and `>` which causes the markdown to render funny

Reply:
OK

Reply:
Oops, I forgot that these render ok in the docs on the website, just ugly on github, this was probably fine how it was (no big deal either way I think though?)